docker-compose.yml

```
@@ -39,6 +39,15 @@ services:
       WORDPRESS_DB_NAME: hurumap
       WORDPRESS_DB_USER: hurumap
       WORDPRESS_DB_PASSWORD: hurumap
+      WORDPRESS_CONFIG_EXTRA: |
+        /* Multisite */
+        define('WP_ALLOW_MULTISITE', true );
+        define('MULTISITE', true);
+        define('SUBDOMAIN_INSTALL', false);
+        define('DOMAIN_CURRENT_SITE', 'localhost');
+        define('PATH_CURRENT_SITE', '/');
+        define('SITE_ID_CURRENT_SITE', 1);
+        define('BLOG_ID_CURRENT_SITE', 1);
```

@kilemensi This doesn't work, and it was the reason why we get the errors.
Instead, it should be then after the network has been installed we can ssh into the volume and add the remaining config.
But Isaiah said, it worked for him after he changed the version from 3 to 3.7
```suggestion
      WORDPRESS_CONFIG_EXTRA: |
        /* Multisite */
        define('WP_ALLOW_MULTISITE', true );
```

---

> But Isaiah said, it worked for him after he changed the version from 3 to 3.7
So have you tried changing to `3.7` @KobbyMmo

---

@kilemensi, @esirK  method didn't work for me.
cc @nerdlet , @KhadijaMahanga

---

Cool @nerdlet / @gertie-sheshe can you try with `WORDPRESS_CONFIG_EXTRA` changed to just
```yml
      WORDPRESS_CONFIG_EXTRA: |
        /* Multisite */
        define('WP_ALLOW_MULTISITE', true );
```
and let us know? It could be we're using different versions of docker/docker-compose so if ☝🏾 works for everyone, we'll change it to that.

---

@kilemensi After the above change  ☝️ it works